addons/web/static/tests/views/fields/datetime_field.test.js

```diff
@@ -503,3 +503,31 @@ test("datetime field: use picker with arabic numbering system", async () => {
     await animationFrame();
     expect(queryFirst("[name=datetime] input")).toHaveValue("٠٢/٠٨/٢٠١٧ ١١:٤٥:٠٠");
 });
+
+test("list datetime with date widget test", async () => {
```

Comment:
this test isn't enough

We want to assert that:
 - a datetime with the show_time option set to false doesn't display the time in readonly
 - when, in a editable list view, the row is switched in edition, the time is displayed

There's no need to have a date/datetime field with widget date (and no need to specify widget=datetime on the datetime field, only the option)